Bixal/move.mil
# Error box and other messages cleanup v2

This PR completes the fix for Pivotal bug 158978968 - error message in locator map tool looks funky. <s>It might be worth refactoring the formatting of these boxes in general at some future time, but this at least patches an acute issue with single-line messages for now.</s> **UPDATE: this more sustainable fix is now integrated**

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Edits the rendered markup inside a Locator map code component to match that needed for warning/callout boxes to appear as expected.
- Edits the warning/callout paragraph markup similarly to extend the change to all boxes.
- Tweaks the css to allow the boxes to look well-aligned for both single-line and multi-line text whether within page content or within tools.

## Testing

To verify the changes proposed in this pull request…

1. Pull code
1. Compile theme assets and rebuild tools
1. Go to the locator map page
1. Click "Search By Your Location"
1. DENY browser access to your location
1. See the error message that appears below. Both the text and the icon to its left should be vertically centered.
1. Go to `/moving-guide/oconus` (as an example) and look at the info box just below the page intro text. Both the text and the icon to its left should be vertically centered.

## Screenshots

### Locator map warning - local fixed
<img width="1060" alt="screen shot 2018-08-02 at 1 30 39 pm" src="https://user-images.githubusercontent.com/29130580/43601678-c6d581ec-965b-11e8-82a7-f3f76e7ca11d.png">

### OCONUS page content - local fixed
<img width="841" alt="screen shot 2018-08-02 at 1 31 17 pm" src="https://user-images.githubusercontent.com/29130580/43601705-dad76f0c-965b-11e8-9862-7bf7db7eee79.png">

### Service-specific info page - bigger box example with change
<img width="569" alt="screen shot 2018-08-02 at 1 54 55 pm" src="https://user-images.githubusercontent.com/29130580/43601723-e96b81d4-965b-11e8-81f7-176964325533.png">
